pkg/pss/pss.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package pss
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/bee/pkg/tags"
+	"github.com/ethersphere/bee/pkg/trojan"
+)
+
+// Pss is the top-level struct, which takes care of message sending
+type Pss struct {
+	storer     storage.Storer
+	tags       *tags.Tags
+	handlers   map[trojan.Topic]Handler
+	handlersMu sync.RWMutex
+	//metrics metrics
+	//logger  logging.Logger
+}
+
+// Monitor is used for tracking status changes in sent trojan chunks
+type Monitor struct {
+	// returns the state of the trojan chunk that is being monitored
+	State chan tags.State
+}
+
+// NewPss inits the Pss struct with the localstore
+func NewPss(localStore storage.Storer, tags *tags.Tags) *Pss {
```

Comment:
the name suggests `localStore` but type is store interface. If it can be only localstore, then the type should be specific.

Reply:
this is wrong, I agree I will change it. good point.

Reply:
fixed